src/foam/nanos/approval/ApprovalRequest.js

```diff
@@ -668,9 +682,23 @@
             })
             .catch((err) => {
               console.warn(err.message || err);
-              return false;
+              if ( self.refObjId && self.refDaoKey && self.__subContext__[self.refDaoKey_] ) {
+                property = self.__subContext__[self.refDaoKey].of.ID;
+                objId = property.adapt.call(property, self.refObjId, self.refObjId, property);
+                return self.__subContext__[self.refDaoKey]
+                  .find(objId)
+                  .then((obj) => {
```

Comment:
Does it still work without this block?

Reply:
.then((obj) => { return !! obj; }), I mean, which could also have been written as just:
.then(obj => !!obj)

Reply:
thanks, updating to `.then(obj => !! obj)` works